civihr_default_theme/scss/components/_datepicker.scss

```diff
@@ -86,3 +86,13 @@ div[uib-datepicker-popup-wrap] {
     border-color: $input-border;
   }
 }
+
+// Styles for the datepicker form civcrm
```

Comment:
then put this in a separate partial (`_datepicker-civicrm.scss`?) leaving a comment on top explain what this specific datepicker is

then you rename this partial (`_datepicker.scss`) so it specifies the nature of this other datepicker. I guess it's angular? So call it something like `_datepicker-angular.scss`